Essentials/src/com/earth2me/essentials/commands/Commandspawnmob.java

```diff
@@ -59,4 +60,13 @@ public void run(final Server server, final CommandSource sender, final String co
         final User target = getPlayer(ess.getServer(), args, 2, true, false);
         SpawnMob.spawnmob(ess, server, sender, target, mobParts, mobData, mobCount);
     }
+
+    @Override
+    protected List<String> getTabCompleteOptions(Server server, User user, String commandLabel, String[] args) {
+        if (args.length == 1) {
+            return Lists.newArrayList(SpawnMob.mobParts(args[0]));
+        } else {
+            return java.util.Collections.emptyList();
```

Comment:
To be consistent with the rest of the Essentials codebase, I suggest reverting this back to not using a fully qualified package name.